### ISSUE TITLE
Can PF6 really be used as output for CCL LUT 3?

### ISSUE THREAD

**Issue by DaleMitchell** [SpenceKonde/DxCore#107]
The PF6 pin is noted to be input only in the datasheets from Microchip; however, the pinout diagrams in this GitHub repository show PF6 to be an alternate pin for CCL LUT 3 output.

**Comment by SpenceKonde** [comment]
Nope.. Maybe on the DD series when those finally arrive., but not the DA/DB series

**Comment by SpenceKonde** [comment]
Fixed. Normally I summon @MCUdude  (he does the pinout diagrams for me) - but I was able to do this myself. I'll have to make sure that this change is communicated to him when I get changes made to them inthe future - I do need to have him adapt them to show that the core supports PWM all of thje TCA mappings, and on PA4/5/6/7 for TCD . for the other mappings of the Type A's you just have the same symbols, but lighten the color so it looks faded or l;ike a disabled option, which it is. But how to express in the diagram that from TCD you get PA4 and PA5, PA5 and PA6, PA6 and PA7 or PA 4 and PA7, but if you try to have PA4 and PA6 or PA5 and PA7 on at the same time, setting either of them changes duty cycle of the other too.
